src/styles/components/forms/_checkbox.scss

```diff
@@ -12,23 +12,13 @@
     margin-top: -4px; // vertically center box
     width: 20px;
     height: 20px;
-    background: $mc-color-gray-100;
+    background: var(--mc-theme-checkbox-fb-bg);
```

Comment:
let's kill `-fb` in these vars, they are essentially just an implementation detail